2022/10/19 13:01:32 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "js/jquery-1.12.4.min.js". in H:\OpenServer\domains\framework\web\CWebApplication.php:286
Stack trace:
#0 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('js/jquery-1.12....')
#1 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/jquery-1.12.4.min.js
HTTP_REFERER=http://auth.net/gii/default/login
---
2022/10/19 13:01:36 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "js/jquery-1.12.4.min.js". in H:\OpenServer\domains\framework\web\CWebApplication.php:286
Stack trace:
#0 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('js/jquery-1.12....')
#1 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/jquery-1.12.4.min.js
HTTP_REFERER=http://auth.net/gii/default/index
---
2022/10/19 13:01:39 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "js/jquery-1.12.4.min.js". in H:\OpenServer\domains\framework\web\CWebApplication.php:286
Stack trace:
#0 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('js/jquery-1.12....')
#1 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/jquery-1.12.4.min.js
HTTP_REFERER=http://auth.net/gii/form/index
---
2022/10/19 13:01:41 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "js/jquery-1.12.4.min.js". in H:\OpenServer\domains\framework\web\CWebApplication.php:286
Stack trace:
#0 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('js/jquery-1.12....')
#1 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/jquery-1.12.4.min.js
HTTP_REFERER=http://auth.net/gii/default/index
---
2022/10/19 13:01:43 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "js/jquery-1.12.4.min.js". in H:\OpenServer\domains\framework\web\CWebApplication.php:286
Stack trace:
#0 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('js/jquery-1.12....')
#1 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/jquery-1.12.4.min.js
HTTP_REFERER=http://auth.net/gii/model/index
---
2022/10/19 13:04:17 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "js/jquery-1.12.4.min.js". in H:\OpenServer\domains\framework\web\CWebApplication.php:286
Stack trace:
#0 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('js/jquery-1.12....')
#1 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/jquery-1.12.4.min.js
HTTP_REFERER=http://auth.net/gii/model/index
---
2022/10/19 13:04:18 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "js/jquery-1.12.4.min.js". in H:\OpenServer\domains\framework\web\CWebApplication.php:286
Stack trace:
#0 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('js/jquery-1.12....')
#1 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/jquery-1.12.4.min.js
HTTP_REFERER=http://auth.net/gii/model/index
---
2022/10/19 13:12:23 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "asmlkas". in H:\OpenServer\domains\framework\web\CWebApplication.php:286
Stack trace:
#0 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('asmlkas')
#1 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/asmlkas
---
2022/10/19 14:05:10 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "login". in H:\OpenServer\domains\framework\web\CController.php:483
Stack trace:
#0 H:\OpenServer\domains\framework\web\CController.php(270): CController->missingAction('login')
#1 H:\OpenServer\domains\framework\web\CWebApplication.php(282): CController->run('login')
#2 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('auth/login')
#3 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/auth/login
---
2022/10/19 14:05:27 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "login". in H:\OpenServer\domains\framework\web\CController.php:483
Stack trace:
#0 H:\OpenServer\domains\framework\web\CController.php(270): CController->missingAction('login')
#1 H:\OpenServer\domains\framework\web\CWebApplication.php(282): CController->run('login')
#2 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('auth/login')
#3 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/auth/login
---
2022/10/19 14:05:37 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "login". in H:\OpenServer\domains\framework\web\CController.php:483
Stack trace:
#0 H:\OpenServer\domains\framework\web\CController.php(270): CController->missingAction('login')
#1 H:\OpenServer\domains\framework\web\CWebApplication.php(282): CController->run('login')
#2 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('auth/login')
#3 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/auth/login
HTTP_REFERER=http://auth.net/
---
2022/10/19 14:32:48 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "logout". in H:\OpenServer\domains\framework\web\CController.php:483
Stack trace:
#0 H:\OpenServer\domains\framework\web\CController.php(270): CController->missingAction('logout')
#1 H:\OpenServer\domains\framework\web\CWebApplication.php(282): CController->run('logout')
#2 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('auth/logout')
#3 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/auth/logout
HTTP_REFERER=http://auth.net/
---
2022/10/19 14:37:03 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "login". in H:\OpenServer\domains\framework\web\CController.php:483
Stack trace:
#0 H:\OpenServer\domains\framework\web\CController.php(270): CController->missingAction('login')
#1 H:\OpenServer\domains\framework\web\CWebApplication.php(282): CController->run('login')
#2 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('auth/login')
#3 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/auth/login
HTTP_REFERER=http://auth.net/
---
2022/10/19 14:53:27 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '9222' for column 'create_time' at row 1. The SQL statement executed was: INSERT INTO `user` (`username`, `password`, `email`, `create_time`, `update_time`, `last_activity`, `role`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6).
2022/10/19 14:53:27 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '9222' for column 'create_time' at row 1 in H:\OpenServer\domains\framework\db\CDbCommand.php:358
Stack trace:
#0 H:\OpenServer\domains\framework\db\ar\CActiveRecord.php(1080): CDbCommand->execute()
#1 H:\OpenServer\domains\framework\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 H:\OpenServer\domains\auth.net\protected\controllers\AuthController.php(39): CActiveRecord->save()
#3 H:\OpenServer\domains\framework\web\actions\CInlineAction.php(49): AuthController->actionRegister()
#4 H:\OpenServer\domains\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 H:\OpenServer\domains\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 H:\OpenServer\domains\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 H:\OpenServer\domains\framework\web\CWebApplication.php(282): CController->run('register')
#8 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('auth/register')
#9 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/auth/register
HTTP_REFERER=http://auth.net/auth/register
---
2022/10/19 17:05:24 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "login". in H:\OpenServer\domains\framework\web\CController.php:483
Stack trace:
#0 H:\OpenServer\domains\framework\web\CController.php(270): CController->missingAction('login')
#1 H:\OpenServer\domains\framework\web\CWebApplication.php(282): CController->run('login')
#2 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#3 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/site/login
---
2022/10/19 17:06:01 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "login". in H:\OpenServer\domains\framework\web\CController.php:483
Stack trace:
#0 H:\OpenServer\domains\framework\web\CController.php(270): CController->missingAction('login')
#1 H:\OpenServer\domains\framework\web\CWebApplication.php(282): CController->run('login')
#2 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#3 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/site/login
---
2022/10/19 17:06:53 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "login". in H:\OpenServer\domains\framework\web\CController.php:483
Stack trace:
#0 H:\OpenServer\domains\framework\web\CController.php(270): CController->missingAction('login')
#1 H:\OpenServer\domains\framework\web\CWebApplication.php(282): CController->run('login')
#2 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#3 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/site/login
---
2022/10/19 17:06:54 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "login". in H:\OpenServer\domains\framework\web\CController.php:483
Stack trace:
#0 H:\OpenServer\domains\framework\web\CController.php(270): CController->missingAction('login')
#1 H:\OpenServer\domains\framework\web\CWebApplication.php(282): CController->run('login')
#2 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#3 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/site/login
---
2022/10/19 17:11:03 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in H:\OpenServer\domains\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 H:\OpenServer\domains\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(WebUser), 'You are not aut...')
#1 H:\OpenServer\domains\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 H:\OpenServer\domains\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 H:\OpenServer\domains\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 H:\OpenServer\domains\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 H:\OpenServer\domains\framework\web\CController.php(291): CFilterChain->run()
#6 H:\OpenServer\domains\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 H:\OpenServer\domains\framework\web\CWebApplication.php(282): CController->run('')
#8 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('test')
#9 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/test
---
2022/10/19 17:17:05 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in H:\OpenServer\domains\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 H:\OpenServer\domains\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(WebUser), 'You are not aut...')
#1 H:\OpenServer\domains\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 H:\OpenServer\domains\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 H:\OpenServer\domains\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 H:\OpenServer\domains\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 H:\OpenServer\domains\framework\web\CController.php(291): CFilterChain->run()
#6 H:\OpenServer\domains\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 H:\OpenServer\domains\framework\web\CWebApplication.php(282): CController->run('edit')
#8 H:\OpenServer\domains\framework\web\CWebApplication.php(141): CWebApplication->runController('test/edit')
#9 H:\OpenServer\domains\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 H:\OpenServer\domains\auth.net\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/test/edit
---
